src/utils/scores.ts

```diff
@@ -0,0 +1,82 @@
+import { BigQuery } from '@google-cloud/bigquery';
+
+import { PROJECT } from '@/config';
+
+const bigqueryClient = new BigQuery({ projectId: PROJECT });
+
+export async function getIssueEventsForTeam(team) {
```

Comment:
Don't see a good way to test this specifically 😔

Reply:
You mean mocking out bigquery? Could we wrap it in an interface that only has methods for getting the stuff we need from BigQuery, then mock that out?

Reply:
Another idea is just to create a spy that verifies that the exact SQL we expect gets sent, and provides mock results.

Reply:
Ended up just creating a spy that verifies the exact sql